distributed/scheduler.py

```diff
@@ -4448,13 +4450,15 @@ def update_graph(
 
         spans_ext: SpansExtension | None = self.extensions.get("spans")
         if spans_ext:
-            span_annotations = spans_ext.new_tasks(new_tasks)
-            if span_annotations:
-                resolved_annotations["span"] = span_annotations
-            else:
-                # Edge case where some tasks define a span, while earlier tasks in the
-                # same TaskGroup don't define any
-                resolved_annotations.pop("span", None)
+            # observe_tasks does not necessarily contain all runnable tasks;
```

Comment:
Did you mean `new_tasks`?
```suggestion
            # new_tasks does not necessarily contain all runnable tasks;
```